formapi/test_app/calls.py

```diff
@@ -28,7 +28,7 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
         # Set the label for the "username" field.
-        self.username_field = get_user_model()._meta.get_field("username")
+        self.username_field = User._meta.get_field("username")
```

Comment:
We should use `django.contrib.auth.get_user_model` here. Otherwise it will break for projectes overriden the user model